frontend/src/app/components/application-top-bar/application-top-bar.component.html

```diff
@@ -30,10 +30,10 @@
         (menuOpened)="menuIsOpen = true"
         [attr.data-testId]="'user-options'"
         [matMenuTriggerFor]="menu"
-        class="topBarEntry flex-nowrap btn"
+        class="top-bar-entry flex-nowrap btn"
       >
-        <mat-icon aria-hidden="false" aria-label="person icon" fontIcon="person_outline" id="topbar-user"></mat-icon>
-        <span [attr.data-testId]="'user-name'" class="fw-normal fs-6 d-none d-md-flex">{{ userFullName }}</span>
+        <mat-icon aria-hidden="false" aria-label="person icon" fontIcon="person_outline" id="top-bar-user"></mat-icon>
+        <span [attr.data-testId]="'username'" class="fw-normal fs-6 d-none d-md-flex">{{ userFullName }}</span>
```

Comment:
to remain consistend user-name should be in kebab case

Reply:
It got defined in [here](https://codimd.puzzle.ch/DWVLjwB8SkaCw0-0LbREWg) as one word